diff --git a/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/a/ext_emconf.php b/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/a/ext_emconf.php
index 2b1821ffb7..0623bdcd4f 100644
--- a/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/a/ext_emconf.php
+++ b/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/a/ext_emconf.php
@@ -12,10 +12,10 @@ $EM_CONF[$_EXTKEY] = [
     'state' => 'stable',
     'uploadfolder' => 0,
     'clearCacheOnLoad' => 1,
-    'version' => '11.5.6',
+    'version' => '11.5.7',
     'constraints' => [
         'depends' => [
-            'typo3' => '11.5.6',
+            'typo3' => '11.5.7',
         ],
         'conflicts' => [],
         'suggests' => [],
diff --git a/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/b/ext_emconf.php b/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/b/ext_emconf.php
index cc2570dbff..0ed0456f46 100644
--- a/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/b/ext_emconf.php
+++ b/typo3/sysext/backend/Tests/Functional/Configuration/SiteConfiguration/Fixtures/Extensions/conf_overriding/b/ext_emconf.php
@@ -12,11 +12,11 @@ $EM_CONF[$_EXTKEY] = [
     'state' => 'stable',
     'uploadfolder' => 0,
     'clearCacheOnLoad' => 1,
-    'version' => '11.5.6',
+    'version' => '11.5.7',
     'constraints' => [
         'depends' => [
-            'typo3' => '11.5.6',
-            'a' => '11.5.6',
+            'typo3' => '11.5.7',
+            'a' => '11.5.7',
         ],
         'conflicts' => [],
         'suggests' => [],
diff --git a/typo3/sysext/backend/composer.json b/typo3/sysext/backend/composer.json
index c6124af5f8..3a30604532 100644
--- a/typo3/sysext/backend/composer.json
+++ b/typo3/sysext/backend/composer.json
@@ -20,8 +20,8 @@
 	},
 	"require": {
 		"psr/event-dispatcher": "^1.0",
-		"typo3/cms-core": "11.5.6",
-		"typo3/cms-recordlist": "11.5.6"
+		"typo3/cms-core": "11.5.*@dev",
+		"typo3/cms-recordlist": "11.5.*@dev"
 	},
 	"suggest": {
 		"typo3/cms-install": "To generate url to install tool in environment toolbar"
diff --git a/typo3/sysext/backend/ext_emconf.php b/typo3/sysext/backend/ext_emconf.php
index d0e80e0e67..f9b65b87bd 100644
--- a/typo3/sysext/backend/ext_emconf.php
+++ b/typo3/sysext/backend/ext_emconf.php
@@ -9,11 +9,11 @@ $EM_CONF[$_EXTKEY] = [
     'author' => 'TYPO3 Core Team',
     'author_email' => 'typo3cms@typo3.org',
     'author_company' => '',
-    'version' => '11.5.6',
+    'version' => '11.5.7',
     'constraints' => [
         'depends' => [
-            'typo3' => '11.5.6',
-            'recordlist' => '11.5.6',
+            'typo3' => '11.5.7',
+            'recordlist' => '11.5.7',
         ],
         'conflicts' => [],
         'suggests' => [],
